pkgs/by-name/i2/i2pd-tools/package.nix

```
@@ -0,0 +1,45 @@
+{
+  lib,
+  stdenv,
+  fetchgit,
+  boost,
+  zlib,
+  openssl,
+}:
+
+stdenv.mkDerivation rec {
+  pname = "i2pd-tools";
+  version = "unstable";
+
+  #tries to access the network during the tests, which fails
+
+  src = pkgs.fetchgit {
+      url = "https://github.com/PurpleI2P/i2pd-tools.git";
+    hash = "sha256-mmCs8AHHKhx1/rDp/Vc1p2W3pufoTa4FcJyJwD919zw=";
+    fetchSubmodules = true;
+  };
+
+  buildInputs = with pkgs; [ zlib.dev openssl.dev boost.dev ];
```

```suggestion
  buildInputs = [ zlib openssl boost ];
```
You also need to import the library individually